File: Second Seminar/SecondSeminar/src/main/java/com/example/SecondSeminar/member/domain/AuthMemberJpaRepository.java
package com.example.SecondSeminar.member.domain;

import java.util.Optional;
import org.springframework.data.jpa.repository.JpaRepository;

public interface AuthMemberJpaRepository extends JpaRepository<AuthMember, Long> {
    Optional<AuthMember> findByNickname(String nickname);
}

File: Second Seminar/SecondSeminarKotlin/src/main/kotlin/com/example/SecondSeminarKotlin/member/domain/AuthMemberJpaRepository.kt
package com.example.SecondSeminarKotlin.member.domain

import org.springframework.data.jpa.repository.JpaRepository
import java.util.*

interface AuthMemberJpaRepository : JpaRepository<AuthMember, Long> {
    fun findByNickname(nickname: String): Optional<AuthMember>
}